roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -14,6 +14,9 @@ confluent:
     metrics_enabled: true
 
 jolokia_enabled: true
+jolokia_auth_mode: none # Possible values: [none, basic]
+jolokia_user: admin
```

Comment:
Not sure we should be setting default username and password, as it could lead to the customer having a false sense of security.  Similar to all the consumer routers that get hacked because they have default credentials.

Reply:
Yes makes sense... unfortunately when we get to the 600-post branch it is important that everything has a default or else the dictionary logic blows up. And we don't do variable assertions in the 55x branches, or in 60x yet. So i decided to keep in line w the patterns of the project. And we will have to document that when jolokia_auth_mode == basic, jolokia_user and jolokia_password must be set. 

Similarly there are default user/passwords for the sasl_plain install that have this same issue. I'm sure there are more

Reply:
Makes sense.  Maybe something to think about for 6.1 or later.